rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -522,6 +553,223 @@ void save_callsFederationSupportSave() throws IOException {
         }
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("svp tests")
+    class SvpTests {
+        private Federation activeFederation;
+        private Federation proposedFederation;
+
+        private BtcTransaction svpFundTransactionUnsigned;
+        private Sha256Hash svpFundTransactionHashUnsigned;
+
+        private Transaction rskTx;
+        private Keccak256 rskTxHash;
+
+        private PegoutsWaitingForConfirmations pegoutsWaitingForConfirmations;
+
+        @BeforeEach
+        void setUp() {
+            rskExecutionBlock = mock(Block.class);
+            long rskExecutionBlockNumber = 1000L;
+            when(rskExecutionBlock.getNumber()).thenReturn(rskExecutionBlockNumber);
+
+            activeFederation = FederationTestUtils.getGenesisFederation(federationConstantsMainnet);
+            federationSupport = mock(FederationSupport.class);
+            when(federationSupport.getActiveFederation()).thenReturn(activeFederation);
+            when(federationSupport.getActiveFederationAddress()).thenReturn(activeFederation.getAddress());
+
+            List<UTXO> utxos = BitcoinTestUtils.createUTXOs(10, activeFederation.getAddress());
```

Comment:
```suggestion
            List<UTXO> activeFederationUTXOs = BitcoinTestUtils.createUTXOs(10, activeFederation.getAddress());
```
Let's use meaningful variable names